modules/node/handler/node_handler.e

```diff
@@ -124,7 +133,16 @@ feature -- HTTP Methods
 						view_response.set_node (l_node)
 						view_response.set_revision (l_rev)
 						view_response.execute
+					elseif
+						attached current_user (req) as l_user and then
```

Comment:
I am wondering if this could not use a permission here.
Since usually we don't need in the code to check if current user is admin.

What is the requirement here?

If auth user is author of node (or admin) then he can "See" the node? or node revision?
otherwise ... this is a not found answer?
This looks weird, and maybe I am tired, but this would deserve a comment to explain that.